lib/imperative-test.js

```diff
@@ -447,6 +447,31 @@ class ImperativeTest extends Test {
     });
   }
 
+  resolvesTest(input, expected, check) {
```

Comment:
I'd prefer to use `resolves` and `rejects` accordingly.

Reply:
What about `test.isResolved()` and `test.isRejected()` (similar to `test.isError()`)?

Reply:
I think the names @nechaido proposed may lead to confusion similar to `test.error()` and `test.isError()`.